lib/breakers/uptime_middleware.rb

```diff
@@ -49,7 +49,9 @@ def outage_response(outage:, service:)
     end
 
     def handle_request(service:, request_env:, current_outage: nil)
+      start_time = Time.now
       return @app.call(request_env).on_complete do |response_env|
+        response_env[:duration] = Time.now - start_time
```

Comment:
Can we use the best practice for custom headers that are not part of HTTP standard? I believe custom headers are usually prefixed with `X-`

https://www.safaribooksonline.com/library/view/restful-web-services/9780596809140/ch01s13.html#I_programlisting1_d1e2475

That being said, https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Field_names says:

> Non-standard header fields were conventionally marked by prefixing the field name with X-[2] but this convention was deprecated in June 2012 because of the inconveniences it caused when non-standard fields became standard.[3] An earlier restriction on use of Downgraded- was lifted in March 2013